packages/ladle/lib/shared/types.ts

```diff
@@ -280,3 +280,20 @@ export type GetUserViteConfig = {
 export type EntryData = {
   [key: string]: ParsedStoriesResult;
 };
+
+export type MetaJson<M = any> = {
+  about: {
+    homepage: string;
+    github: string;
+    version: number;
+  };
+  stories: { [key: string]: MetaJsonStory<M> };
+};
+
+export type MetaJsonStory<M = any> = {
+  name: string;
+  levels: string[];
+  meta: M;
```

Comment:
Can this be made stricter, an extension of this type https://github.com/tajo/ladle/blob/main/packages/ladle/lib/app/exports.ts#L138

Reply:
Yeah this is what I wanted to do, but it's a bit awkward since Meta is defined in the app exports but these types are in the shared types.

I would just move the Meta interface over into the shared types and re-export it but I get a conflict with this Meta const:
<img width="729" alt="image" src="https://github.com/user-attachments/assets/55bbaf60-39b3-4664-bde2-31e8e93ee1a5">

I think I have something that should work though, so let's see what you think.

Reply:
@tajo How about this implementation?

Reply:
lgtm